src/docs/framework-integration/angular.md

```diff
@@ -98,9 +133,13 @@ export const config: Config = {
 };
 ```
 
-> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
+> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the
```

Comment:
it might be good to call out above that `componentCorePackage` has to match whatever name you gave the stencil component library.

If the user has entered the commands exactly as in the tutorial this will be `"stencil-library"`, so maybe we could change the value to that so it 'just works' ?